subsys/net/openthread/rpc/Kconfig

```diff
@@ -31,7 +31,7 @@ config OPENTHREAD_RPC_CLIENT
 
 config OPENTHREAD_RPC_SERVER
 	bool "OpenThread over RPC server"
-	depends on NET_L2_OPENTHREAD
+	depends on NET_L2_OPENTHREAD || ZTEST
```

Comment:
You mean enable `CONFIG_OPENTHREAD_RPC_SERVER` via CMake instead of Kconfig? But then you have to manually configure everything that is normally configured via this Kconfig. Isn't this less maintainable?

Reply:
This was just a suggestion. If you think the current implementation is better, just leave it `as is`.